lib/utils/ProductListQueryConfiguration.dart

```diff
@@ -6,15 +6,19 @@ import 'package:openfoodfacts/utils/ProductFields.dart';
 class ProductListQueryConfiguration extends AbstractQueryConfiguration {
   final List<String> barcodes;
 
+  /// See [AbstractQueryConfiguration.languages] for
+  /// parameter's description.
   ProductListQueryConfiguration(
     this.barcodes, {
     final OpenFoodFactsLanguage? language,
+    final List<OpenFoodFactsLanguage> languages = const [],
```

Comment:
Unfortunately I don't think we can deprecate `lc` from there.

Reply:
I don't understand. Could you please elaborate?

Reply:
I guess I should have selected the line below, with `String lc`.
`lc` is deprecated in the abstract class but I don't think there's a way to tag it as deprecated in the child classes as well. Perhaps dart/flutter does it automagically: "hey you're using a parameter that is used as is in the parent class and is deprecated there"?